### DISCUSSION TITLE
Slim, temple 0.10.1, NoMethodError

### DISCUSSION THREAD

**Discussion by olleolleolle** [sinatra/sinatra#1925] [General] [answered]
Hello, I made this upgrade https://my.diffend.io/gems/temple/0.10.0/0.10.1

...and my Slim templates failed to render in my tiny Sinatra app.

Error:

> NoMethodError: undefined method `empty?' for nil:NilClass

**Workaround**: In order to live out this day, I pinned the temple dependency to `= 0.10.0` in my Gemfile.

```
[GEM_ROOT]/gems/sinatra-3.0.6/lib/sinatra/base.rb:947 :in `__tilt_9660`
      @response = Response.new
      @pinned_response = nil
      template_cache.clear if settings.reload_templates

      invoke { dispatch! }
```

https://github.com/sinatra/sinatra/blob/v3.0.6/lib/sinatra/base.rb#L947

Did you suffer the same fate?

**Comment by dentarg** [comment]
I don't think I have any apps using slim so I have not seen this. Can you supply a repro?

**Comment by dentarg** [comment]
[Looks](https://github.com/sinatra/sinatra/actions/runs/4985378856/jobs/8924907611#step:4:73) like CI is using `temple 0.8.2`, not sure why. A pull request making it use something newer would be cool.

**Comment by dentarg** [comment, marked as answer]
Connecting this thread with the issue you created over at https://github.com/judofyr/temple/issues/145
